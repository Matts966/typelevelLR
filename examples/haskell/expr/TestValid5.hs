
import Expr

main :: IO ()
main = print $ begin |> lp |> lp |> lp |> num 1 |> rp |> rp |> rp |> end
