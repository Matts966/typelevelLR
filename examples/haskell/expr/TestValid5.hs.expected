TToE (FToT (Paren (TToE (FToT (Paren (TToE (FToT (Paren (TToE (FToT (Num 1)))))))))))
